File: Swift/1.swift
import Foundation

func getNaturalLog()->Double
{
    var index:Double = 1
    var item:Double = 1
    var result:Double = 0

    while (item >= 1e-4) {
        result += item
        item /= index++
    }

    return result
}

print(String(getNaturalLog()))

File: Java/1.java
public class NaturalLog {

	public static void main(String args[]){
	    System.out.println(getNaturalLog());
	  }
	
	static double getNaturalLog()
    {
        int index = 1;
        double item = 1;
        double result = 0;

        while (item >= 1e-4)
        {
            result += item;
            item /= index++;
        }

        return result;
    }
}
